tests/integration/components/categories-list-test.js

```diff
@@ -30,11 +41,11 @@ test('it renders the categories and sorts them by name', function(assert) {
   ];
 
   this.set('categories', categories);
-  this.render(hbs`{{categories-list categories=categories}}`);
+  page.render(hbs`{{categories-list categories=categories}}`);
 
-  assert.equal(this.$('.categories-list').length, 1);
-  assert.equal(this.$('.category-item').length, 3);
-  assert.equal(this.$('.category-item:eq(0)').text().trim(), 'Alphabets');
-  assert.equal(this.$('.category-item:eq(1)').text().trim(), 'Society');
-  assert.equal(this.$('.category-item:eq(2)').text().trim(), 'Zoology');
+  assert.ok(page.isRendered);
```

Comment:
I know this was in the tests before, but seems like we can remove if we're testing other things. What do you think @dmoon1221?